inst/rmarkdown/templates/ajs_outbreak/skeleton/skeleton.Rmd

```diff
@@ -1581,126 +1890,160 @@ mortality_rate(deaths$deaths, deaths$population, multiplier = 10000) %>%
-## - [ ] Choose what variable you would like to fill (for polygons) or colour (for points) with (counts or AR)
-## - [ ] DELETE OR COMMENT OUT THE SECTION YOU WONT BE USING
+## Create a categories of ARs by region ----------------------------------------
 
-## create a categorical variable for plotting 
-max_ar    <- max(ar$`Upper 95%CI`, na.rm = TRUE) # define your highest AR
+# define maximum
+max_ar    <- max(ar_region$`Upper 95%CI`, na.rm = TRUE) # define your highest AR
 
+# create groups - one group will be 0 only, then up to 4 more groups evenly
+# divided to maximum
 breakers <- as.integer(c(0, # include zero as a standalone group
              seq(1, max_ar, by = max_ar/4) # 1 to maximum with four divisions
              ))
-## add a categorical variable using the age_categories function 
-## nb in this case we arent using ages - but it functions the same way!
-ar <- mutate(ar, 
+
+## create a categorical variable using the age_categories function 
+## (we aren't using ages - but it functions the same way!)
+ar_map <- mutate(ar_region, 
              categories = age_categories(`AR (per 10,000)`, 
-                                         breakers = breakers)
-             )
+                                         breakers = breakers))
+
 
 
-## join your ar or case counts based on matching shapefile names with regions in your ar table
+## Join table to shapefile -----------------------------------------------------
 mapsub <- left_join(map, ar, by = c("name" = "Region"))
 
 
-## plotting with polygons ------------------------------------------------------
 
-## choropleth 
+## Plot for polygons -----------------------------------------------------------
+
 ## you could also fill by cases using `Cases (n)` in the fill option instead of `AR (per 10,000)`
 ggplot() +
-  geom_sf(data = mapsub, aes(fill = categories), col = "grey50") + # shapefile as polygon
-  coord_sf(datum = NA) + # needed to avoid gridlines being drawn
-  annotation_scale() + # add a scalebar
+  # shapefile as polygon
+  geom_sf(data = mapsub, aes(fill = categories), col = "grey50") + 
+  
+  # needed to avoid gridlines being drawn
+  coord_sf(datum = NA) + 
+  
+  # add a scalebar
+  annotation_scale() + 
+  
   # color the scale to be perceptually uniform 
   # drop FALSE keeps all levels 
   # name allows you to change the legend title 
   scale_fill_brewer(drop = FALSE, palette = "OrRd", name = "AR (per 10,000)") + 
-  geom_sf_text(data = mapsub, aes(label = name), colour = "grey50") + # label polygons
-  theme_void() # remove coordinates and axes
+  
+  # label polygons
+  geom_sf_text(data = mapsub, aes(label = name), colour = "grey50") + 
+  
+  # remove coordinates and axes
+  theme_void() 
 
 
 
-## plotting with points --------------------------------------------------------
+## Plot for points -------------------------------------------------------------
 
 # ggplot() +
-#   # plot shapefile as point 
+#   # plot shapefile as point
```

Comment:
I'm wondering if this is necessary (it's one of those old code chunks that was commented out and never really tested. It has the unfortunate property of not actually plotting points, but basically just drawing a very thick border around the regions:

![image](https://user-images.githubusercontent.com/3639446/64124467-fe827480-cd9e-11e9-9826-ada5ce624efc.png)

Reply:
That's cus there aren't long and lat collected in the dictionary (which is designed for hospital settings) 
But I think we should still keep it in (in a seperate chunk) - because some community settings will still be collecting GPS data.

Reply:
As long as we know that the construct works.

Reply:
Relatively convinced it was brought forward directly from the older generic template which had coords. 
Easily tested by taking a sample of coords from within the polygon ... Using sf::st_sample

Reply:
that's a thing I did not know!

Reply:
Edit: I take it back that won't work for plotting points of cases as the cases longlat would come from the linelist not the shapefile

Reply:
yeah, i don't think you can have an underlying point shapefile. you need at least some sort of polygon case map. but then R would have to process the lat/lon into the regions, which i'm not sure how to do. i would say delete the point sections.

Reply:
but that doesn't make sense to me -- we can't do chloropleths with points. and then they'll just be floating in the ether without a basemap.

Reply:
you plot a blank polygon with geom_sf and on top you plot geom_point using lat long from the linelist (on training webstie will eventually show how to have openstreetmap basemaps)

Reply:
> * think we need to be consistent with spacing between lines of related code. E.g. in some place we have blank lines between layers of a pipe or ggplot, in others not. Indifferent which one use, lets just stick to one though

tried to be as consistent as possible, but for one of the pipe sections (first use of tab_univariate), it's difficult to read as one big "chunk". so left with extra spacing for extra readability.